db/migrate/20230120234843_creates_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatesPosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :author, foreign_key: true, index: true
```

Comment:
- [ ] I'm happy to see that the name of a user will be called author as shown on the ERD diagram, but we should let Ruby know which table is an author we are referring to by explicitly adding `foreign_key: { to_table: 'users' }`